gradle/dependency-locks/embulkPluginRuntime.lockfile

```diff
@@ -1,7 +1,24 @@
 # This is a Gradle generated file for dependency locking.
 # Manual edits can break the build and are not advised.
 # This file is expected to be part of source control.
+com.fasterxml.jackson.core:jackson-annotations:2.6.7
+com.fasterxml.jackson.core:jackson-core:2.6.7
+com.fasterxml.jackson.core:jackson-databind:2.6.7
+com.fasterxml.jackson.datatype:jackson-datatype-guava:2.6.7
+com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.6.7
+com.google.guava:guava:18.0
+com.ibm.icu:icu4j:54.1.1
 commons-codec:commons-codec:1.10
+commons-io:commons-io:2.7
 commons-logging:commons-logging:1.2
```

Comment:
We may want to find where this `commons-logging` come from, and replace it with `jcl-over-slf4j:1.7.12`. See https://github.com/embulk/embulk-output-gcs/pull/44 for reference.